### PR TITLE
Lint the code according to PEP8

### PR DESCRIPTION
**Pull request by rht**

### CODE REVIEW

**Comment by rht** [comment]
It appears that the failed assertion test here is statistical https://travis-ci.org/bfelbo/DeepMoji/builds/299427333#L3750. Sometimes the assertion is satisfied.

**Comment by bfelbo** [comment]
Okay, I've made that assertion test less strict. Thanks for making me aware and implementing PEP8. Can you change it such that we only enforce PEP8 on python files? For instance, the readme does not need to be PEP8 compliant :)

**Comment by rht** [comment]
Hmm, according to the [flake8 doc](http://flake8.pycqa.org/en/latest/#using-flake8), it should accept a directory as an argument, which in the case of https://github.com/bfelbo/DeepMoji/pull/18/commits/768a751c36972fe9c9ce814630d3716ef24b6ada is the current directory. There is no need of extra regex to the argument such as `**/*.py`. I'm not sure if `flake8` checks for python file through the file's mime type or just its extension though (most likely the latter).

**Comment by bfelbo** [comment]
Hmm.. flake8 seems to assume that all files in the provided directory should be forced to follow PEP8. A good way to avoid enforcing PEP8 on the readme etc. would likely be to only run flake8 on the directories `deepmoji`, `examples`, `scripts`, `tests`. Could you make this change?

**Comment by rht** [comment]
I did the change: https://travis-ci.org/bfelbo/DeepMoji/jobs/317566410. Redoing it since I forgot to remove `--exclude='env/'`.

File: deepmoji/create_vocab.py [hunk 1]
```diff
@@ -113,15 +118,17 @@ def populate_master_vocab(self, vocab_path, min_words=1, force_appearance=None):
 
             sizes[path] = sum(dicts[path].values())
             print('Overall word count for {} -> {}'.format(path, sizes[path]))
-            print('Overall word number for {} -> {}'.format(path, len(dicts[path])))
+            print(
+                'Overall word number for {} -> {}'.format(path, len(dicts[path])))
```
Comment (bfelbo) [inline deepmoji/create_vocab.py]: The line wrapping in PEP8 is a bit excessive.. This does not seem more readable. Could you disable the line wrapping in flake8 and rerun it?

**Comment by bfelbo** [comment]
Thanks! The line wrapping in PEP8 is a bit excessive.. Some of the lines being wrapped are definitely less readable than before. Could you disable the line wrapping in flake8 and rerun it?

**Comment by rht** [comment]
Right, gotcha, fixed+rebased!

**Comment by bfelbo** [comment]
That was quick! I've just noticed that it's messing up the README.md files inside the script folders. Is there an exclude option to not process these files?

**Comment by rht** [comment]
I tried `autopep8 --exclude examples/README.md ...` / `autopep8 --exclude README.md ...` with the README.md file still being written. I have manually not committed the change to the commit. Should be good to go.

**Comment by bfelbo** [comment]
Okay, thanks!

**Pull request merged**